### PR TITLE
Modified triage replied

### PR DESCRIPTION
**Pull request by john-westcott-iv**
##### SUMMARY

Split no progress into issue and pr
Added community.general standard response

##### ISSUE TYPE

 - Bug or Docs Fix

##### COMPONENT NAME

 - Other

##### AWX VERSION

```
awx: 21.2.1.dev50+geb7414d2b6.d20220706
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: .github/triage_replies.md [hunk 1]
```diff
@@ -51,6 +63,10 @@ Thank you once again for this and your interest in AWX!
 ### Code of Conduct
 - Hello. Please keep in mind that Ansible adheres to a Code of Conduct in its community spaces. The spirit of the code of conduct is to be kind, and this is your friendly reminder to be so. Please see the full code of conduct here if you have questions: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html 
 
+### Community.general
+- Hello. The awx-ee contains the collections and dependencies needed for supported AWX features to function. Anything beyond that (like the community.general package) will require you to build your own EE. For information on how to do that, see https://ansible-builder.readthedocs.io/en/stable/ \
+\
+Also just a heads up that the Ansible Community team is looking at building an EE that corresponds to all of the collections inside the ansible package. That may help you if and when it happens.
```
Comment (shanemcd) [inline .github/triage_replies.md]: @gundalow do you know if there's a link to a proposal for this?

Comment (john-westcott-iv) [reply .github/triage_replies.md]: Good catch, copy paste did nothing with the inline link. In the PR you closed you had linked https://github.com/ansible-community/ansible-build-data/blob/main/6/ansible.in. Do we want to include this sentence or should we just remove it?

Comment (gundalow) [reply .github/triage_replies.md]: https://github.com/ansible-community/community-topics/issues/31

File: .github/triage_replies.md [hunk 2]
```diff
@@ -51,6 +63,10 @@ Thank you once again for this and your interest in AWX!
 ### Code of Conduct
 - Hello. Please keep in mind that Ansible adheres to a Code of Conduct in its community spaces. The spirit of the code of conduct is to be kind, and this is your friendly reminder to be so. Please see the full code of conduct here if you have questions: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html 
 
+### Community.general
```
Comment (gundalow) [inline .github/triage_replies.md]: Why is the title `community.general`, that's the name of a collection.

Comment (john-westcott-iv) [reply .github/triage_replies.md]: I just thought it would be an easy name to find when looking through the file, opened to suggestions.

Comment (gundalow) [reply .github/triage_replies.md]: General Community?

Comment (john-westcott-iv) [reply .github/triage_replies.md]: EE Requirements?

File: .github/triage_replies.md [hunk 3]
```diff
@@ -51,6 +63,10 @@ Thank you once again for this and your interest in AWX!
 ### Code of Conduct
 - Hello. Please keep in mind that Ansible adheres to a Code of Conduct in its community spaces. The spirit of the code of conduct is to be kind, and this is your friendly reminder to be so. Please see the full code of conduct here if you have questions: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html 
 
+### EE Contents / Community General
+- Hello. The awx-ee contains the collections and dependencies needed for supported AWX features to function. Anything beyond that (like the community.general package) will require you to build your own EE. For information on how to do that, see https://ansible-builder.readthedocs.io/en/stable/ \
+\
+Also just a heads up that the Ansible Community team is looking at building an EE that corresponds to all of the collections inside the ansible package. That may help you if and when it happens; see https://github.com/ansible-community/ansible-build-data/ for details.
```
Comment (gundalow) [inline .github/triage_replies.md]:
```suggestion
The Ansible Community is looking at building an EE that corresponds to all of the collections inside the ansible package. That may help you if and when it happens; see https://github.com/ansible-community/community-topics/issues/31 for details.
```

Comment (john-westcott-iv) [reply .github/triage_replies.md]: Updated.

**Review by gundalow — APPROVED** [review]
Great, thanks

**Pull request merged**